SPECS/privacyidea-server.spec

```diff
@@ -125,16 +125,13 @@ fi
 
 ###################################################
 # The webserver
-# first we need to start the webserver to let it create the self-signed certificates
-systemctl start httpd
-# mkdir -p /var/run/wsgi
+# first we need to create the self-signed certificates on CentOS 8.
+[[ -x /usr/libexec/httpd-ssl-gencerts ]] && /usr/libexec/httpd-ssl-gencerts
```

Comment:
What would happen during an update, if the certificate files have been manually updated with productive certificates, while the privacyidea.conf has not been touched in the setup.
Would the certificate files be overwritten by this?

Reply:
Replay this senario with a untouched privacyidea.conf, and a modifyed certificate. Certificate was not changed.
Second senario with changes in privacyidea.conf and a other certificate. Both was not overwritten.  Pi-Update create a new  file: privacyidea.conf.rpmnew how assume.

Reply:
Also tested in centos8 with the same result as above.

Reply:
@pablo-knight  Do you merge it?

Reply:
merged